fedoracommunity/consumers.py

```diff
@@ -84,9 +105,81 @@ def consume(self, msg):
                     kwargs = make_kwargs(connector, path, info, filters)
                     lookup_key = generator(**kwargs)
                     hashed_key = mangler(lookup_key)
-                    log.info("Updating %s" % lookup_key)
-                    log.debug("    hash %s" % hashed_key)
+                    log.info("Refreshing %s" % lookup_key)
                     # Destroy the old value
                     self.mc.delete(hashed_key)
                     # Run the connector to re-fill the cache.
                     fn(*args, **kwargs)
+                    log.info(" Done with %s" % hashed_key)
+
+    def update_xapian(self, msg):
+        # If any number of different pkgdb things happen to a package, let's
+        # just update and not care too much about whatever it was that just
+        # happened.
+        if '.pkgdb.' not in msg['topic']:
+            return
+
+        # This one is spammy
+        if '.pkgdb.acl.update' in msg['topic']:
```

Comment:
I dunno.. but I thought it would be safer to be explicit:

- I really don't want to needlessly update the db when people change ACLs *en masse*.
- The code below really can't handle something that doesn't have a ``package_listing`` entry.